api/lib/mno_enterprise/concerns/controllers/jpi/v1/products_controller.rb

```diff
@@ -0,0 +1,25 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::ProductsController
+  extend ActiveSupport::Concern
+
+  #==================================================================
+  # Included methods
+  #==================================================================
+  # 'included do' causes the included code to be evaluated in the
+  # context where it is included rather than being executed in the module's context
+  included do
+    respond_to :json
```

Comment:
Is it needed? If not present at the `Jpi::V1::BaseResourceController` level we should put it there